docs/source/gbd2017_models/risks/correlated_risks/index.rst

```diff
@@ -0,0 +1,1199 @@
+
+Risk exposure correlation
+------------------------------
+
+The proposed method to introduce correlation in risk factor exposure is by finding the Spearman’s correlation coefficient between two or more risks and sampling from a multivariate normal distribution when assigning a position on an exposure distribution. This plot shows data from a joint distribution reflecting the variance and covariance between height and weight at one month using data from a cohort study. The top panel shows the values for the height and weight and the bottom panel shows the percentiles within the distributions (propensity values).
+
+.. image:: gaussian_copulae.jpg
+
+Please see the ipynb Notebook for an example of how we intend to code this. 
+												 
+Attributable fraction correlation
+-------------------------------------
+Suppose that we introduce correlation in risk exposure but don’t properly adjust for the changes this causes in the combined attributable fractions between the two or more risks. The rate of disease among individuals and in aggregate will be wrong and the amount of error increases with larger values of the combined attributable fractions. 
+
+With some testing and simulation, we are able to show that the population attributable fraction for multiple risk factors depends on the correlation in those risks. To date, Vivarium has been using a risk deleted rate (e.g. incidence or mortality) based on an assumption that the risk exposures are independent. However, the more correlated the risks exposures, the more wrong this assumption will be. The overall idea is to take GBD rates and remove the combined contribution of risk factor attributable fractions (PAFs):
+ 
+.. math::
+	i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
+ 
+Where i_0 is the risk-deleted rate, PAF is the population attributable fraction for a single or multiple risk factors, and  i_GBDis the rate from GBD. A combined PAF for multiple risk factors is the product of 1 minus the independent PAFs. 
+ 
+.. math::
+	1 - \text{PAF}_{\text{Multiplicative}} = \left(1 - \text{PAF}_{\text{Risk1}}\right)\cdot\left(1 - \text{PAF}_{\text{Risk2}}\right)
+	
+This can be written in a slightly different way for each individual in the simulation and what we are calling the Multiplicative approach to combined PAFs:
+ 
+.. math::
+	PAF_{multiplicative} = 1 - (1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n}RR_1^{e1_i}}) \cdot (1 - \frac{1}{\frac{1}{n}\sum_{i=1}^{n}RR_2^{e2_i}})
```

Comment:
As noted in the call today, this equation is wrong: There should be no "1 - " inside the two pairs of parentheses -- it should just be 1 over the average RR.

As we also noted in the call, it would be good to clarify how it works to compute the average RR's using 1/n at the population level vs. using the exposures for each category.

And it would also be good to show how this equation is a special case of the general joint PAF equation under the assumption that the risk factors are independent, as James does in one of the pdf files he posted in Slack.